src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -282,7 +286,8 @@ public void checkMainTools() {
   }
 
   public void checkWikiMarkupTools() {
-    for (int i = 1, j = i + 1; i < 21; i++) {
+    int wikitextShortcutsWikiMarkupSection = 21; //edit tools 'Wiki markup' section in MediaWiki:Edittools
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ATrailingCommentCheck)